src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ColorPickerTests/WinUIColorPickerPage.xaml

```diff
@@ -11,229 +11,234 @@
 
```

Comment:
This test page is copy-pasted from upstream WinUI. I definitely would keep their indentation. Here this again avoids all the lines showing as changed.